warehouse/ingest-scripts/src/main/resources/bin/system/cleanup_old_pdsh_logs.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Cleanup old pdsh logs on the cluster
+
+if [[ `uname` == "Darwin" ]]; then
+    THIS_SCRIPT=`python -c 'import os,sys;print os.path.realpath(sys.argv[1])' $0`
+else
+    THIS_SCRIPT=`readlink -f $0`
+fi
+THIS_DIR="${THIS_SCRIPT%/*}"
+cd ${THIS_DIR}
+
+. ../ingest/ingest-env.sh
```

Comment:
cleanup_old_files_util.sh already sources ingest-env, so you can avoid sourcing it twice.

Reply:
Fixed in next commit.